pkg/ingress/controller/controller.go

```diff
@@ -338,17 +340,6 @@ type Api6Controller struct {
 	Stop                      <-chan struct{}
 }
 
-func (api6 *Api6Controller) ApisixRoute(controller *Controller) {
```

Comment:
We discussed earlier that we should keep the v1 version for a while, declare in the document "deprecated", and delete the logic of v1 in version 1.0.

Reply:
Sorry I forgot to set this PR to draft, the v1 ApisixRoute will be kept of course.